ui/src/treeDataProvider/AzureTreeDataProvider.ts

```diff
@@ -220,6 +221,25 @@ export class AzureTreeDataProvider<TRoot = ISubscriptionRoot> implements IAzureT
         return element.parent;
     }
 
+    public async getSubscriptionPromptStep(wizardContext: Partial<types.ISubscriptionWizardContext>): Promise<types.AzureWizardPromptStep<types.ISubscriptionWizardContext> | undefined> {
+        const subscriptions: AzureTreeItem<ISubscriptionRoot>[] = await this.ensureRootTreeItems();
+        if (subscriptions.length === 1) {
+            Object.assign(wizardContext, subscriptions[0].root);
+            return undefined;
```

Comment:
There are more properties in ISubscriptionRoot than ISubscriptionWizardContext (tenantId/userId).  Is this assignment a good idea?  Should ISubscriptionWizardContext extend ISubscriptionRoot?

Reply:
> Should ISubscriptionWizardContext extend ISubscriptionRoot?

I'm hesitant to do that for this PR because it would affect other repos (If they create a wizardContext, they'll now be required to specify those extra props). For now, I made sure to only copy necessary props